### DISCUSSION TITLE
Need quick jumpstart

### DISCUSSION THREAD

**Discussion by BerndSon** [jolution/todo-nukem#8] [Q&A] [answered]
Hello, I installed the VSCode extension and need jumpstart help to use it. When I insert "// TODO", for example, how can I activate the different options? Maybe there is a conflict with other extensions (sonarlint, Copilot)?

**Comment by juliankasimir** [comment]
Hello @BerndSon,

Thank you for reaching out and for testing the alpha version of our plugin! We greatly appreciate your test and feedback.

Regarding your question, in [supported languages](https://github.com/jolution/todo-nukem-snippet-vscode?tab=readme-ov-file#-supported-languages), you can type "todo" or "fixme" and then press the Tab key to activate the different options.

Does this work for you?

At the moment, i don't know about any conflicts with other extensions like SonarLint or Copilot.

If you encounter any further issues or have any more questions, please don't hesitate to let us know. Your input helps us improve the plugin for everyone.


Thanks again for being a valuable contributor and tester!

**Comment by juliankasimir** [comment, marked as answer]
Hello @BerndSon,

Thank you for pointing out that the suggestion is not activated within the comments. This appears to be a setting that is on by default in Vscode.

I have provided instructions for this in the [FAQ](https://github.com/jolution/todo-nukem-snippet-vscode?tab=readme-ov-file#faq).

```
"editor.quickSuggestions": {
    "comments": true,
    "strings": true
}
```

I would be happy if this helps. And if so, please mark the answer as correct.
